src/cmd/linuxkit/run_vcenter.go

```diff
@@ -90,7 +92,7 @@ func runVcenter(args []string) {
 	}
 
 	// Test any passed in files before creating a new VM
-	checkFile(*newVM.path)
+	// checkFile(*newVM.path)
```

Comment:
thanks @rn 

I temporarily commented out because I am not sure of the specification of the `linuxkit run vcenter`. Is it supposed to push as well the image to vCenter?